src/auth/signup/components/SignUp.jsx

```diff
@@ -4,6 +4,8 @@ import { connect } from 'react-redux';
 import signUpUser from '../redux/actions/signUpAction';
 import validation from '../Validation';
 import '../CSS/signup.scss';
+import FacebookButton from '../../socialAuth/facebook/components/Facebook';
```

Comment:
Are these changes relevant to the story of this PR?

Reply:
Yes they are because if you check develop they are not there.

Reply:
How is social auth related to pagination?

Reply:
The social-auth button on the develop signup were not functional, you can check it out, and this is the fix for them.

Reply:
If they're not doing that mean you make changes under a story that is in no way related to social auth?

Reply:
Am sorry about that, I should have sought advice on handling that.

Reply:
What's the better way to handle that?

Reply:
I guess it is to seek guidance on how to handle it.

Reply:
Please ask some of the team members, I need to be confident that you really understand how to handle it...

Reply:
Yes @AGMETEOR I can and let me work on that.

Reply:
I have removed it all and fixed this problem.